src/openrct2-ui/input/KeyboardShortcuts.h

```diff
@@ -127,14 +32,114 @@ namespace OpenRCT2
 
     namespace Input
     {
+        enum class Shortcut
+        {
+            CloseTopMostWindow,
+            CloseAllFloatingWindows,
+            CancelConstructionMode,
+            PauseGame,
+            ZoomViewOut,
+            ZoomViewIn,
+            RotateViewClockwise,
+            RotateViewAnticlockwise,
+            RotateConstructionObject,
+            UndergroundViewToggle,
+            RemoveBaseLandToggle,
+            RemoveVerticalLandToggle,
+            SeeThroughRidesToggle,
+            SeeThroughSceneryToggle,
+            InvisibleSupportsToggle,
+            InvisiblePeopleToggle,
+            HeightMarksOnLandToggle,
+            HeightMarksOnRideTracksToggle,
+            HeightMarksOnPathsToggle,
+            AdjustLand,
+            AdjustWater,
+            BuildScenery,
+            BuildPaths,
+            BuildNewRide,
+            ShowFinancialInformation,
+            ShowResearchInformation,
+            ShowRidesList,
+            ShowParkInformation,
+            ShowGuestList,
+            ShowStaffList,
+            ShowRecentMessages,
+            ShowMap,
+            Screenshot,
+
+            // New
+            ReduceGameSpeed,
+            IncreaseGameSpeed,
+            OpenCheatWindow,
+            RemoveTopBottomToolbarToggle,
+            ScrollMapUp,
+            ScrollMapLeft,
+            ScrollMapDown,
+            ScrollMapRight,
+            OpenChatWindow,
+            QuickSaveGame,
+            ShowOptions,
+            MuteSound,
+            WindowedModeToggle,
+            ShowMultiplayer,
+            PaintOriginalToggle,
+            DebugPaintToggle,
+            SeeThroughPathsToggle,
+            RideConstructionTurnLeft,
+            RideConstructionTurnRight,
+            RideConstructionUseTrackDefault,
+            RideConstructionSlopeDown,
+            RideConstructionSlopeUp,
+            RideConstructionChainLiftToggle,
+            RideConstructionBankLeft,
+            RideConstructionBankRight,
+            RideConstructionPreviousTrack,
+            RideConstructionNextTrack,
+            RideConstructionBuildCurrent,
+            RideConstructionDemolishCurrent,
+            LoadGame,
+            ClearScenery,
+            GridlinesDisplayToggle,
+            ViewClipping,
+            HighlightPathIssuesToggle,
+            TileInspector,
+            AdvanceToNextTick,
+            SceneryPicker,
+            ScaleUp,
+            ScaleDown,
+            InsertCorruptElement,
+            CopyElement,
+            PasteElement,
+            RemoveElement,
+            MoveElementUp,
+            MoveElementDown,
+            IncreaseXCoord,
+            DecreaseXCoord,
+            IncreaseYCoord,
+            DecreaseYCoord,
+            IncreaseElementHeight,
+            DecreaseElementHeight,
+            ToggleClearanceChecks,
+
+            Count,
+
+            Undefined = 0xFFFF,
+        };
+
+        constexpr size_t GetShortcutsCount()
```

Comment:
I'd have to check but I thought in general we did a namespace with a constexpr value of type size_t:
```
constexpr size_t ShortcutsCount = static_cast<size_t>(Shortcut::Count);
```